skiko/src/commonMain/cpp/generated/PathSegmentIterator.cc

```diff
@@ -22,40 +22,32 @@ SKIKO_EXPORT KNativePointer org_jetbrains_skia_PathSegmentIterator__1nGetFinaliz
 }
 
 
-SKIKO_EXPORT KInteropPointer org_jetbrains_skia_PathSegmentIterator__1nNext
-  (KNativePointer ptr) {
-    TODO("implement org_jetbrains_skia_PathSegmentIterator__1nNext");
-}
-     
-#if 0 
-SKIKO_EXPORT KInteropPointer org_jetbrains_skia_PathSegmentIterator__1nNext
-  (KNativePointer ptr) {
+SKIKO_EXPORT void org_jetbrains_skia_PathSegmentIterator__1nNext
+  (KNativePointer ptr, float* points) {
```

Comment:
Better use KInt* here, just for consistency, and recover from bits on Kotlin side.